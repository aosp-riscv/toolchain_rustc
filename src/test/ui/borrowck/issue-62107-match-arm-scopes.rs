fn main() {
    let e: i32;
    match e {
<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
        //~^ ERROR use of possibly uninitialized variable
=======
        //~^ ERROR use of possibly-uninitialized variable
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)
        ref u if true => {}
        ref v if true => {
            let tx = 0;
            &tx;
        }
        _ => (),
    }
}
