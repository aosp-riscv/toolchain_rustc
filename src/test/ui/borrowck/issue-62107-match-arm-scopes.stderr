<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
error[E0381]: use of possibly uninitialized variable: `e`
  --> $DIR/issue-62107-match-arm-scopes.rs:3:11
   |
LL |     match e {
   |           ^ use of possibly uninitialized `e`
=======
error[E0381]: use of possibly-uninitialized variable: `e`
  --> $DIR/issue-62107-match-arm-scopes.rs:3:11
   |
LL |     match e {
   |           ^ use of possibly-uninitialized `e`
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)

error: aborting due to previous error

For more information about this error, try `rustc --explain E0381`.
